oci/BUILD.bazel

```diff
@@ -40,30 +40,31 @@ pkg_tar(
 )
 
 pkg_tar(
-    name = "pachd_tar",
+    name = "pachd_host_tar",
     srcs = ["//src/server/cmd/pachd"],
     owner = NONROOT,
     ownername = NONROOT_USERNAME,
 )
 
+platform_transition_filegroup(
```

Comment:
This makes `bazel build //:pachd_image` put a Linux pachd in the container, not a whatever-your-current-OS-is pachd.